package.json

```diff
@@ -24,5 +24,10 @@
     "husky": "^8.0.3",
     "lint-staged": "^13.2.2",
     "prettier": "2.8.8"
+  },
+  "dependencies": {
+    "dotenv": "^16.0.3",
+    "openai": "^3.2.1",
+    "process": "^0.11.10"
```

Comment:
where are you using process package? process.env is not an npm package, its a built in obj that nodejs gives us. remove it